country-currency-map.js

```diff
@@ -26,7 +27,20 @@ function formatCurrency(value, currencyAbbr) {
     return `${value} ${currencyAbbr}`;
 }
 
+function getCurrencyList() {
+    return currencyMap;
+}

```

Comment:
Possibly? I didn't know how to order it... alphabetically? Or with usd at top? and I wanted to keep the details about each currency in what we return (symbol, abbr, etc.), so I didn't just want to return a list of strings. But if you think i should change it from a single object to an array of objects, that could be possible.